db/seed_SDC_json.js

```diff
@@ -66,54 +68,85 @@ let platformTable = [
 
 
 
-function writeJsonData(cb) {
-    let i = 1;
+function writeJsonData(id) {
+    let i = id;
     let msIn10Years = 1000 * 60 * 60 * 24 * 365 * 10;
 
 
-    while (i < 10000000) {
-        let platformIds = [1, 2, 3];
 
-        let obj = {
-            id: i,
-            description: faker.fake('{{lorem.paragraph}} {{lorem.paragraph}}'),
-            release_date: new Date(Date.now() - Math.floor(Math.random() * msIn10Years)).toISOString(),
-            developers: [],
-            publishers: []
+    let platformIds = [1, 2, 3];
+
+    let obj = {
+        id: i,
+        description: faker.fake('{{lorem.paragraph}} {{lorem.paragraph}}'),
+        release_date: new Date(Date.now() - Math.floor(Math.random() * msIn10Years)).toISOString(),
+        developers: [],
+        publishers: []
+
+    }
 
-        }
+    let randomNumber = Math.ceil(Math.random() * 3);
 
-        let randomNumber = Math.ceil(Math.random() * 3);
+    for (let j = 0; j < randomNumber; j++) {
 
-        for(let j = 0; j<randomNumber;j++){
-            
-            let randIdx = Math.floor(Math.random() * platformIds.length);
-            let randomPlatform = platformIds[randIdx];
-            platformIds.splice(randIdx, 1);
+        let randIdx = Math.floor(Math.random() * platformIds.length);
+        let randomPlatform = platformIds[randIdx];
+        platformIds.splice(randIdx, 1);
 
-            let copyCompanyTable = Array.from(companyTable)
+        let copyCompanyTable = Array.from(companyTable)
 
-            let randomDevObj = copyCompanyTable[Math.ceil(Math.random() * 50)-1];
-            let randomPubObj = copyCompanyTable[Math.ceil(Math.random() * 50)-1];
+        let randomDevObj = copyCompanyTable[Math.ceil(Math.random() * 50) - 1];
+        let randomPubObj = copyCompanyTable[Math.ceil(Math.random() * 50) - 1];
 
-            
-            randomDevObj['platform'] = platformTable[randomPlatform-1];
-            randomPubObj['platform'] = platformTable[randomPlatform-1];
 
-            obj.developers.push(randomDevObj);
-            obj.publishers.push(randomPubObj);
+        randomDevObj['platform'] = platformTable[randomPlatform - 1];
+        randomPubObj['platform'] = platformTable[randomPlatform - 1];
 
-        }
-        data.push(obj);
-        i++;
+        obj.developers.push(randomDevObj);
+        obj.publishers.push(randomPubObj);
 
     }
 
-    cb();
+    return obj
+
+
+
+}
+
+
+function writeTenMillionJson(writer,encoding,callback){
```

Comment:
I feel like this function could use a bit of commenting.

Reply:
thanks for that suggestion, ill add comment on it!